### PR TITLE
feat: feature registry integration in sdk and react libraries

### PR DESCRIPTION
**Pull request by luzzif**

### CODE REVIEW

File: packages/react/src/hooks/useTemplateFeatureEnabledFor copy.ts
Comment (guerrap) [inline packages/react/src/hooks/useTemplateFeatureEnabledFor copy.ts:1]: `copy` leftover in the filename.

File: packages/react/src/hooks/useTemplateFeatureEnabledFor copy.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    templateId?: number;
+    featureId?: number;
+    account?: Address;
+}
+
+export function useTemplateFeatureEnabledFor(
+    params?: TemplateFeatureEnabledForParams,
+): {
+    loading: boolean;
+    allowed: boolean;
```
Comment (guerrap) [inline packages/react/src/hooks/useTemplateFeatureEnabledFor copy.ts:17]: I'd rename `allowed` to `enabled` to be consistent with the function name, or the opposite, wdyt?

File: packages/react/src/hooks/useTemplateFeatureEnabledFor copy.ts
Comment (luzzif) [reply packages/react/src/hooks/useTemplateFeatureEnabledFor copy.ts:1]: Addressed in 5aa06bf3f48a5d76554a5d8b1d9bec736be0605d

File: packages/react/src/hooks/useTemplateFeatureEnabledFor copy.ts (hunk 1 above)
Comment (luzzif) [reply packages/react/src/hooks/useTemplateFeatureEnabledFor copy.ts]: Should be addressed in 3bb57910d803184cb4385d2b2b7f1707c3413123

File: packages/sdk/src/fetcher/subgraph/index.ts [hunk 2]
```diff
@@ -468,6 +471,66 @@ class Fetcher implements IPartialCarrotFetcher {
...
+        subgraphURL: string,
+        managerAddress: Address,
+        templateId: number,
+        featureId: number,
+        account: Address,
+    ): Promise<boolean> {
+        const lowerCaseAccount = account.toLowerCase();
+        const response = await query<GetTemplateFeatureEnabledForQueryResponse>(
+            subgraphURL,
+            GetOracleTemplateFeatureEnabledForQuery,
```
Comment (guerrap) [inline packages/sdk/src/fetcher/subgraph/index.ts:484]: This query should change if the manager address is the KPI token manager or the oracle manager, no?

Comment (luzzif) [reply packages/sdk/src/fetcher/subgraph/index.ts]: Oof big miss from me. Addressed in 80c567ec6b274177517a70863efa42d9b8200444

**Review by guerrap — APPROVED** [review]

**Pull request merged**